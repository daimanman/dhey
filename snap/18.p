{"C":10,"Z":"2","N":10,"Remark":"TEST TASK","Url":"http://www.baidu.com","Method":"post","TaskId":18,"Type":"","FileId":"3","Status":0,"Err":"","QH":{}}