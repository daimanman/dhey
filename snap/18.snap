{"AvgTotal":1.1851919999999998,"Fastest":0.1143078,"Slowest":0.1213266,"Average":0.11851919999999998,"Rps":82.42215639439331,"AvgConn":0.05841098,"AvgDNS":0.000401,"AvgReq":0,"AvgRes":0.00010024,"AvgDelay":0.060007979999999995,"ConnMax":0,"ConnMin":0.0010025,"DnsMax":0,"DnsMin":0.0010025,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0571552,"DelayMin":0.0620767,"Lats":[0.1143078,0.1163135,0.1163135,0.1163135,0.1193213,0.1193213,0.1193213,0.1213266,0.1213266,0.1213266],"ConnLats":[0,0,0,0,0.0010025,0.0010025,0.0010025,0,0,0.0010025],"DnsLats":[0,0,0,0,0.0010025,0.0010025,0.0010025,0,0,0.0010025],"ReqLats":[0,0,0,0,0,0,0,0,0,0],"ResLats":[0,0,0,0,0,0,0,0.0010024,0,0],"DelayLats":[0.0571552,0.0591609,0.0591609,0.0591609,0.0600714,0.0600714,0.0600714,0.0610743,0.0620767,0.0620767],"Total":121326600,"ErrorDist":{},"StatusCodeDist":{"404":10},"SizeTotal":5640,"SizeReq":564,"NumRes":10,"LatencyDistribution":[{"Percentage":10,"Latency":0.1163135},{"Percentage":25,"Latency":0.1163135},{"Percentage":50,"Latency":0.1193213},{"Percentage":75,"Latency":0.1213266},{"Percentage":90,"Latency":0.1213266},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1143078,"Count":1,"Frequentcy":0.1},{"Mark":0.11500968,"Count":0,"Frequentcy":0},{"Mark":0.11571156,"Count":0,"Frequentcy":0},{"Mark":0.11641344,"Count":3,"Frequentcy":0.3},{"Mark":0.11711532000000001,"Count":0,"Frequentcy":0},{"Mark":0.11781720000000001,"Count":0,"Frequentcy":0},{"Mark":0.11851908,"Count":0,"Frequentcy":0},{"Mark":0.11922096,"Count":0,"Frequentcy":0},{"Mark":0.11992284,"Count":3,"Frequentcy":0.3},{"Mark":0.12062472,"Count":0,"Frequentcy":0},{"Mark":0.1213266,"Count":3,"Frequentcy":0.3}],"TaskId":18,"Duration":0.1213266}